middleware/unauthorizedHandler.js

```diff
@@ -7,7 +7,7 @@
 */
 module.exports = function() {
   return (err, req, res, next) => {
-    if (err.statusCode === 401) {
+    if (err.statusCode === 401 || err.status === 401) {
```

Comment:
From the docs: 

> When an error is written, the following information is added to the response:
> - The res.statusCode is set from err.status (or err.statusCode). If this value is outside the 4xx or 5xx range, it will be set to 500.
> - The res.statusMessage is set according to the status code.

If the default error handler has already processed this by the time this middleware got called, does that mean that instead of this line with 2 inputs we can use `res.statusCode`? cc @stevehobbsdev

Reply:
> The res.statusCode is set from err.status (or err.statusCode)

Yep 👍 makes sense

Reply:
Would probably fix the codecov audit too